.ci/deps.sh

```diff
@@ -37,9 +37,28 @@ case $CIRCLE_BUILD_IMAGE in
     deps="$deps libxml2-utils php-codesniffer"
     # Add extra infer deps
     deps_infer="$deps_infer ocaml camlp4"
+    # Note that g++ & gfortran 4.9 or higher are also needed for lintr
+    # dependency Rcpp and 5.x is already included due to other dependencies.
+    # However, it needs activating; see immediately below.
     ;;
 esac
 
+# Activate g++ & gfortran 4.9+ for lintr
+ls -al /usr/bin/gcc /usr/bin/g++ /usr/bin/gfortran || true
```

Comment:
Wait, I just realized that the CircleCI build for this PR at https://circleci.com/gh/jayvdb/coala-bears/347
Doesnt have 4.9 nor 5 nor 6. It says:
```
+ ls -al /usr/bin/gcc /usr/bin/g++ /usr/bin/gfortran
lrwxrwxrwx 1 root root  7 Apr  7  2014 /usr/bin/g++ -> g++-4.8
lrwxrwxrwx 1 root root  7 Apr  7  2014 /usr/bin/gcc -> gcc-4.8
lrwxrwxrwx 1 root root 12 Apr  7  2014 /usr/bin/gfortran -> gfortran-4.8
+ '[' -x /usr/bin/gfortran-6 ']'
+ '[' -x /usr/bin/gfortran-5 ']'
+ '[' -x /usr/bin/gfortran-4.9 ']'
+ '[' -n xenial ']' ...
```